tests/unit/dataset/loader/test_tracelab_trace.py

```
@@ -0,0 +1,1038 @@
+        assert m.take(-1) == []
+        assert m.take(1) == [1]
+
+
+# ---------------------------------------------------------------------------
+# Round timing and ordering
+# ---------------------------------------------------------------------------
+
+
+class TestRoundTiming:
+    def test_api_time_spans_submission_to_last_output(self):
+        submitted, api = round_timing(make_row(submitted=0.0, responded=2.5))
```

Ruff (0.16.1)

[warning] Unpacked variable `submitted` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)